autoload/lsp/signature.vim

```diff
@@ -92,9 +92,15 @@ export def SignatureHelp(lspserver: dict<any>, sighelp: any): void
   if sig->has_key('parameters') && sighelp->has_key('activeParameter')
     var params_len = sig.parameters->len()
     if params_len > 0 && sighelp.activeParameter < params_len
-      var label: string = sig.parameters[sighelp.activeParameter].label
-      hllen = label->len()
-      startcol = text->stridx(label)
+      // label = string | [uint, uint]
```

Comment:
1, '//' is wrong comment string;
2, '[unit, unit]' was that really matching the lsp specification?